rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import typescript from '@rollup/plugin-typescript'
+import pkg from './package.json'
+
+export default [
+  {
+    input: 'dist/index.js',
+    output: {
+      name: 'typescriptMonads',
```

Comment:
Wanna shorten this window name to something more friendly while we are here? `tsm` or `monads` or anything you can think of that is more friendly?

Reply:
`monads` sounds like a good idea. I'll do that.

Reply:
Would need a tweak to the README file as well to whatever name you choose. Thanks!